src/components/Network/index.js

```diff
@@ -231,9 +198,20 @@ class Network extends PureComponent {
   }
 
   renderMovingNodePlaceholder = () => {
-    const { movingNodePlaceholder } = this.state;
-
-    return movingNodePlaceholder && <NodePlaceholder {...movingNodePlaceholder} />;
+    const { movingNode, svgRef } = this.state;
+    const { changeNodePosition } = this.props;
+
+    return movingNode && (
+      <NodeMovingPlaceholder
+        svg={svgRef}
+        node={movingNode}
+        onSetPosition={({ x, y }) => {
```

Comment:
What do you think about extracting this to a function as well so you avoid it to be recreated in every re-render?